src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Runtime.InteropServices;
+using System.IO;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibGpiodDriver : UnixDriver
+    {
+        private SafeChipHandle _chip;
+
+        private Dictionary<int, SafeLineHandle> _pinNumberToSafeLineHandle;
```

Comment:
readonly?

Reply:
Not sure i got this comment correctly, if you mean if this dictionary should be readonly then i don't thinks so, would like to ask why

Reply:
readonly will only mean that the reference cannot be changed not the actual content. IMO this keyword is kinda useless since you can workaround it with reflection and only adds unnecessary complexity like const correctness in C++

Reply:
yes it is thanks, my point is this collection content will be changed even the reference to the collection wouldn't change, doesn't make sense to make readonly